bin/retrieve_dns.py

```diff
@@ -58,9 +60,21 @@ def get_config(config_file):
     c = Configuration()
 
     try:
-        c.gocdb_url = cp.get('auth', 'gocdb_url')
+        c.gocdb_hosts = cp.get('auth', 'gocdb_hosts').split(',')
     except ConfigParser.NoOptionError:
-        c.gocdb_url = None
+        c.gocdb_hosts = None
+
+    try:
+        c.gocdb_pi_cmd = cp.get('auth', 'gocdb_pi_cmd')
+    except ConfigParser.NoOptionError:
+        c.gocdb_pi_cmd = None
+
+    try:
+        c.service_types = cp.get('auth', 'service_types')
+    except ConfigParser.NoOptionError:
+        c.service_types = None
+    if c.service_types and not verify_service_types(c.service_types):
+        raise ValueError("Invalid characters in service type.")
```

Comment:
If `service_types` was missing, or present but empty, in the config - would that cause `c.service_types` to be `None` and a `ValueError` to be raised?

If so, the error message should be change to better explain those cases.

Reply:
ah yes, I must have been confused...